choreolib/src/main/java/choreo/Choreo.java

```diff
@@ -16,14 +16,15 @@
 import choreo.trajectory.SwerveSample;
 import choreo.trajectory.Trajectory;
 import choreo.trajectory.TrajectorySample;
+import choreo.util.AlertUtil;
+import choreo.util.AlertUtil.*;
```

Comment:
Apparently we don't do wildcard imports.
https://github.com/SleipnirGroup/Choreo/pull/997/files/75da3b3e6b15babd09a34c823c95cd91113a5565#diff-429eb6d933fd118960d151d5273731da1a61957530710958ceb39fc48fd6ff86